include/aikido/util/StepSequence.hpp

```diff
@@ -21,8 +19,11 @@ class StepSequence
   /// point. If includeEndpoints is true then the final point in the sequence
   /// will be the end point, even if it is at less than stepSize from the second
   /// to last point.
-  StepSequence(const double _stepSize, const bool _includeEndpoints = true,
-               const double _startPoint = 0.0, const double _endPoint = 1.0);
+  StepSequence(
+      const double _stepSize,
+      const bool _includeEndpoints = true,
+      const double _startPoint = 0.0,
+      const double _endPoint = 1.0);
```

Comment:
It's meaningless to add a `const` qualifier to parameters that are passed by value in a function declaration. Is this something we can enforce with `clang-format`?